File: test_sqrt.c
#include <math.h>
#include <stdio.h>

int main()
{
  int n = 100000;
  long double sum = 0;
  for (int i = 1; i <= n; i++)
    for (int j = 1; j <= n; j++)
      sum += sqrt((long double)i * (long double)j)/((long double)i * (long double)j);
  printf("%Lf\n", sum);

  return 0;
}

File: test_sqrt.lua
function test_sqrt()
  n = 100000
  sum = 0.0
  for i = 1, n do
    for j = 1, n do
      sum = sum + math.sqrt(i * j)/(i * j)
    end
  end
  print(sum)
end

test_sqrt()

